ClosedXML/Excel/Drawings/Style/XLDrawingStyle.cs

```diff
@@ -18,13 +15,50 @@ public XLDrawingStyle()
             Margins = new XLDrawingMargins(this);
             Web = new XLDrawingWeb(this);
         }
+
         //public IXLDrawingFont Font { get; private set; }
         public IXLDrawingAlignment Alignment { get; private set; }
+
         public IXLDrawingColorsAndLines ColorsAndLines { get; private set; }
         public IXLDrawingSize Size { get; private set; }
         public IXLDrawingProtection Protection { get; private set; }
         public IXLDrawingProperties Properties { get; private set; }
         public IXLDrawingMargins Margins { get; private set; }
         public IXLDrawingWeb Web { get; private set; }
+
+        public static IXLDrawingStyle DefaultCommentStyle
+        {
+            get
+            {
+                if (defaultCommentStyle == null)
+                {
+                    defaultCommentStyle = new XLDrawingStyle();
+
+                    defaultCommentStyle
+                        .Margins.SetLeft(0.1)
+                        .Margins.SetRight(0.1)
+                        .Margins.SetTop(0.05)
+                        .Margins.SetBottom(0.05)
+                        .Margins.SetAutomatic()
+                        .Size.SetHeight(59.25)
+                        .Size.SetWidth(19.2)
+                        .ColorsAndLines.SetLineColor(XLColor.Black)
+                        .ColorsAndLines.SetFillColor(XLColor.FromArgb(255, 255, 225))
+                        .ColorsAndLines.SetLineDash(XLDashStyle.Solid)
+                        .ColorsAndLines.SetLineStyle(XLLineStyle.Single)
+                        .ColorsAndLines.SetLineWeight(0.75)
+                        .ColorsAndLines.SetFillTransparency(1)
+                        .ColorsAndLines.SetLineTransparency(1)
+                        .Alignment.SetHorizontal(XLDrawingHorizontalAlignment.Left)
+                        .Alignment.SetVertical(XLDrawingVerticalAlignment.Top)
+                        .Alignment.SetDirection(XLDrawingTextDirection.Context)
+                        .Alignment.SetOrientation(XLDrawingTextOrientation.LeftToRight)
+                        .Properties.SetPositioning(XLDrawingAnchor.Absolute)
+                        .Protection.SetLocked()
+                        .Protection.SetLockText();
+                }
+                return defaultCommentStyle;
```

Comment:
Not exactly what I meant. `IXLDrawingStyle` members have setters, and by changing their properties the user can change the global static default style. Better return a new instance here on every call. This way `DefaultCommentStyle` won't change whatever user does.

Reply:
Of course. Will fix after my beach filled summer holiday.

Reply:
Fixed